src/fungible_token.ts

```diff
@@ -0,0 +1,131 @@
+import { DUST_AMOUNT, hexToString, SignerProvider, TransactionBuilder } from '@alephium/web3'
```

Comment:
Check the naming convention for files in TS. Should be `fungible-token.ts`. Same case in `test/fungible_token.test.ts`